fdbcli/BulkLoadCommand.actor.cpp

```diff
@@ -111,6 +119,11 @@ ACTOR Future<Void> printBulkLoadJobProgress(Database cx, BulkLoadJobState job) {
 	fmt::println("Submitted {} tasks", submitTaskCount);
 	fmt::println("Finished {} tasks", completeTaskCount);
 	fmt::println("Error {} tasks", errorTaskCount);
+	if (totalTaskCount.present()) {
+		fmt::println("Total {} tasks", totalTaskCount.get());
+	} else {
+		fmt::println("Total task count is unknown");
+	}
```

Comment:
Make a method of this if/else since used twice in the one file

Reply:
Will fix this in the next PR.